wgpu/src/lib.rs

```diff
@@ -1036,14 +1066,18 @@ pub struct Operations<V> {
     /// How data should be read through this attachment.
     pub load: LoadOp<V>,
     /// Whether data will be written to through this attachment.
-    pub store: bool,
+    ///
+    /// Note that resolve textures (if specified) are always written to,
+    /// regardless of this setting.
+    pub store: StoreOp,
 }
 
 impl<V: Default> Default for Operations<V> {
+    #[inline]
```

Comment:
Why was this added?

Reply:
counter question, why was it missing - I added this in passing since trivial default like this should imho be inlinable and Rust's compilation model doesn't allow for inlining accros crates unless either this attribute is present or link time optimizations are at play.
Unrelated to this PR ofc, but figured can't hurt. You object?

Reply:
I'm honestly not sure of the full costs/benefits of this - I know it can benefit when you have LTO off (but also why do you have LTO off if you care about performance) but I think there is costs to just slapping it on all over the place, even for trivial stuff.

Reply:
Not a discussion for this PR - let's merge.